### PR TITLE
[#161377001] Two app domains

### PR DESCRIPTION
**Pull request by jonathanglassman**
What
----

Amended content to reflect that the PaaS now has two app domains

How to review
-------------

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. Check that it fulfils https://www.pivotaltracker.com/story/show/161377001 

Who can review
--------------

Anyone except Jon Glassman

### CODE REVIEW

**Comment by keymon** [comment]
If London is now the default region, maybe we should change the examples that refer to Ireland to examples referring to London instead.

**Comment by jonathanglassman** [comment]
Agree

File: source/documentation/deploying_apps/names_routes_domains.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 ### App names and cloudapps.digital hostname clash
```
Comment (keymon) [inline source/documentation/deploying_apps/names_routes_domains.md]: Shall we change the title of this?

Comment (keymon) [inline source/documentation/deploying_apps/names_routes_domains.md]: Shall we change the title of this?

File: source/documentation/deploying_services/configure_cdn.md [hunk 2]
```diff
@@ -57,11 +57,11 @@ When your CDN connects to the `cloudapps.digital` server, the server will presen
 
 #### Configure your CDN
 
-1. Configure your CDN to forward HTTPS requests from your custom domain to the PaaS at the `YOURAPP.cloudapps.digital` domain.
+1. Configure your CDN to forward HTTPS requests from your custom domain to the PaaS at the `YOURAPP.APP_DOMAIN` domain.
 
 2. Configure your CDN to amend the HTTP requests when it forwards those requests by:
 
-  - changing the HTTPS request `Host` header from your custom domain to your app’s domain (for example, change `Host: www.example.com` to `Host: YOURAPP.cloudapps.digital`)
+  - changing the HTTPS request `Host` header from your custom domain to your app’s domain (for example if your app is hosted in the Ireland region, change `Host: www.example.com` to `Host: YOURAPP.london.cloudapps.digital`)
```
Comment (keymon) [inline source/documentation/deploying_services/configure_cdn.md]:
```suggestion
  - changing the HTTPS request `Host` header from your custom domain to your app’s domain (for example if your app is hosted in the London region, change `Host: www.example.com` to `Host: YOURAPP.london.cloudapps.digital`)
```

**Comment by jonathanglassman** [comment]
Thanks @keymon. Please do rebase but don't merge yet until I've checked if it needs tech writer 2i.

**Comment by keymon** [comment]
Rebased.

File: source/documentation/deploying_apps/names_routes_domains.md (hunk 1 above)
Comment (jonathanglassman) [reply source/documentation/deploying_apps/names_routes_domains.md]: Yes

**Comment by jonathanglassman** [comment]
Merging as passed 2i

**Pull request merged**